package.json

```diff
@@ -2,16 +2,28 @@
   "name": "ethereumjs-account",
   "version": "2.0.5",
   "description": "Encoding, decoding and validation of Ethereum's Account schema",
-  "main": "index.js",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/***"
```

Comment:
is this a valid glob? shouldn't it be `dist/**/*`?

Reply:
Ah good catch :)